packages/fast-form-generator-react/src/form/form-item.array.tsx

```diff
@@ -12,6 +12,7 @@ import styles from "./form-item.array.style";
 import { IFormItemArrayClassNameContract } from "../class-name-contracts/";
 import manageJss, { IJSSManagerProps } from "@microsoft/fast-jss-manager-react";
 import { IManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
+import { isDescendant } from "@microsoft/fast-web-utilities";
```

Comment:
Does this actually work? isDescendant was pulled out of fast-web-utilities in favor of just using element.contains()

Reply:
Well, yeah it works or I wouldnt have submitted :)
We'll see what we can do with contains.